### PR TITLE
Edit VILE sprites renamer

### PR DESCRIPTION
**Pull request by Voros2**

### CODE REVIEW

**Comment by chungy** [comment]
This will break Windows compatibility, at least removing the `\` rename, even a Cygwin build can't use `\` in a file name.

Does Windows allow the `[` or `]` characters? I haven't checked, but if it does, we could at least remove those. (And maybe change `\`->`$` to something else so the file name doesn't need escaping on the unix shell)

**Comment by Voros2** [comment]
Windows does support [ and ] last I checked. Reopened PR if it's worth it.

File: src/tools.c [hunk 1]
```diff
@@ -215,6 +215,23 @@ void MakeDir(char file[128], const char *path, const char *dir, const char
...
+   ** replace the VILE\ 
+   ** by          VIL@B 
+   */
+   Normalise(name2,name);
+ 
+   /* 2017-07-17: Only replace backslash. [ and ] characters are 
+      supported in Windows and Unix-derived systems.  */
+   switch(name2[4])
+   {
+     case '\\': name2[4]='@';break;
```
Comment (chungy) [inline src/tools.c]: Perhaps it is a better idea to change this to the `^` character to match the [ZDoom standard in PK3 files](https://zdoom.org/wiki/Using_ZIPs_as_WAD_replacement#Tips).

File: src/tools.c [hunk 2]
```diff
@@ -215,6 +215,23 @@ void MakeDir(char file[128], const char *path, const char *dir, const char
...
+   }
+   switch(name2[6])
+   { 
+     case '\\': name2[6]='@';break;
```
Comment (chungy) [inline src/tools.c]: Here too.

File: src/tools.c [hunk 3]
```diff
@@ -215,6 +215,23 @@ void MakeDir(char file[128], const char *path, const char *dir, const char
...
 {  FILE *fp;
+   char name2[8];  /* AYM 1999-01-13: keep checker happy */
+   /* deal with VILE strange name
+   ** replace the VILE\ 
+   ** by          VIL@B 
+   */
+   Normalise(name2,name);
+ 
+   /* 2017-07-17: Only replace backslash. [ and ] characters are 
+      supported in Windows and Unix-derived systems.  */
```
Comment (chungy) [inline src/tools.c]: Don't bother dating your comment, Git does that just fine automatically ;)

The comment may better just be something like "Replace backslash which is an invalid character on Windows systems." -- no mention of [ or ] are needed.

**Comment by chungy** [comment]
Delete lines 68, 69, 70 of `text.c` too.

**Pull request merged**